src/main/java/nl/knaw/dans/ingest/core/config/ValidateDansBagConfig.java

```diff
@@ -23,6 +28,20 @@ public class ValidateDansBagConfig {
     private URI pingUrl;
 
     private int connectionTimeoutMs;
+    @Valid
+    @NotNull
+    private JerseyClientConfiguration httpClient = new JerseyClientConfiguration();
+
+    @JsonProperty("httpClient")
```

Comment:
When the bean property is already correctly name it is not necessary to explicitly set it with `@JsonProperty`